canonical-websites/www.ubuntu.com
# 2654: Move /about/about-ubuntu/conduct to /community/code-of-conduct

# Note
**Please merge #2685 first**

## Done

- Moved /about/about-ubuntu/conduct to /community/code-of-conduct
- Updated copy according to [copydoc](https://docs.google.com/document/d/1SWBQmBvpVvJnSZVB7aEDWLOo8rBhHgTkYii1Wvgjx20/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/community/code-of-conduct
- Check that it matches the [copydoc](https://docs.google.com/document/d/1SWBQmBvpVvJnSZVB7aEDWLOo8rBhHgTkYii1Wvgjx20/edit)
- Check that /about/about-ubuntu/conduct redirects to /community/code-of-conduct

## Issue / Card

Fixes #2654
